settings.py

```diff
@@ -12,9 +12,19 @@
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
         "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
-    }
+        "TEST": {
+            "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        },
+    },
 }
 
+# DATABASES = {
+#     "default": {
+#         "ENGINE": "django.db.backends.sqlite3",
+#         "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+#     }
+# }
+
 USE_TZ = False
 
 INSTALLED_APPS = ("db",)
```

Comment:
Include one migration in your PR that reflects all changes, do not change `settings.py` file in your solution.